.github/workflows/ci-windows.yml

```diff
@@ -75,6 +74,10 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: ./zlibpng.cmd
+    - name: Build GoogleTest
+      #if: steps.cache-ext.outputs.cache-hit != 'true'
```

Comment:
I think I understand how the cache works now. The cache step is specified as follows:

```
    - name: Cache external dependencies
      id: cache-ext
      uses: actions/cache@v3
      with:
        path: ext
        key: ${{ runner.os }}-${{ hashFiles('ext/*.cmd') }}
```

Note that the key is the hash of all the `ext/*.cmd` files. Since this pull request doesn't modify any `ext/*.cmd` file, we will get a cache hit, but the current cache for Windows doesn't contain a GoogleTest build. This is why we have to comment out the cache hit check in this pull request.

We can uncomment this line when we modify any `ext/*.cmd` file, or afterwards.